### PR TITLE
Update the guideline to markdown

### PR DESCRIPTION
**Pull request by iconstudio**
I replace the text description with a better-looking (I think) readme file. Very small but quite important change.

### CODE REVIEW

File: Langs/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Guidelines
+
+## Requirements
+
+- **Find Your Language Code Here:** https://www.autoitscript.com/autoit3/docs/appendix/OSLangCodes.htm
```
Comment (micwoj92) [inline Langs/README.md:5]: I don't like the bold text here. If you really wanted to use markdown then you could do something like this:
`Find your language code [here](https://www.autoitscript.com/autoit3/docs/appendix/OSLangCodes.htm)`

File: Langs/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- Always use the latest template (0000.lang)
```
Comment (micwoj92) [inline Langs/README.md:6]: Missing dot here, either use them or not, be consistent.

File: Langs/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- Use hex code(number) of your language.
+- Please use your GitHub username as Translator in MetaData section.
+- All strings must be filled out to be accepted.
+- Due to space restrictions, translations do not have to be 1:1.
+- Please do not use slang, due to it being hard to judge/approve.
+- File must be UTF-8 encoded, use GitHub editor and it should take care of that.
+
+## Testing Translations Locally
+1. Go to folder: *%localappdata%\WhyNotWin11\langs*
```
Comment (micwoj92) [inline Langs/README.md:15]: instead of 1. 2. 3. you can use 1. 1. 1. so you won't have to change these numbers each time.

File: Langs/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+3. Edit file and relaunch application
+
+## Submitting
+
+1. Open issue or pull request
+2. In the title please include language name and whether you add a new one or update existing
+
+## Review Process
+
+##### When New Translation
```
Comment (micwoj92) [inline Langs/README.md:26]: I don't like the "when" usage here and below. Also I think it should say translations.
Also, why did you use 5 hashtags here and not three?

**Review by micwoj92 — CHANGES_REQUESTED** [review]
Also the fact that this file was named with 0000- was, so it would show near the template 0000.lang and would be on top when you open the folder on GitHub. Now it requires to scroll. Also this change requires changes to main readme, because the document that you deleted is linked there.

File: Langs/README.md (hunk 4 above)
Comment (iconstudio) [reply Langs/README.md:26]: I tried to make small headers. I don't know why I think that is good idea.

**Comment by iconstudio** [comment]
I fixed the issues, but code quotes can't embed hyperlinks.
Like `[here](https://github.com/rcmaehl/WhyNotWin11/pull/423)`
Like ```[here](https://github.com/rcmaehl/WhyNotWin11/pull/423)```

File: Langs/README.md (hunk 1 above)
Comment (iconstudio) [reply Langs/README.md]: Removed the bold text and placed a embedded hyperlink.

File: Langs/README.md (hunk 2 above)
Comment (iconstudio) [reply Langs/README.md]: Place a dot there.

File: Langs/README.md (hunk 3 above)
Comment (iconstudio) [reply Langs/README.md]: Rewrite lists with 1. 1.

**Comment by micwoj92** [comment]
I know that they can't

**Comment by micwoj92** [comment]
Looks good for me, what do you think @rcmaehl ?

**Review by micwoj92 — APPROVED** [review]

**Review by rcmaehl — APPROVED** [review]

**Pull request merged**